spec/lib/puppet_spec/ssl.rb

```diff
@@ -132,6 +132,11 @@ def self.revoke(serial, crl, ca_key)
       crl
     end
 
+    # def self.create_branched_pki
+    #   root_key = 
+    # end
+
+
```

Comment:
Is this left in deliberately for some reason?

Reply:
nope, this is a mistake.